specifyweb/frontend/js_src/lib/localization/main.ts

```diff
@@ -231,19 +231,7 @@ export const mainText = createDictionary({
     'de-ch': 'Sind Sie sicher, dass Sie diese Seite verlassen wollen?',
   },
   leavePageConfirmationDescription: {
-    'en-us': 'Unsaved changes would be lost if your leave this page.',
```

Comment:
why where the other languages removed?
(the translations looked a good)

also, when en string changes, weblate automatically flags other strings for manual review (and shows to translators how the en string changes, so you certainly don't need to remove the strings in the code

---

and, the original issue mentioned:

> also maybe "would" should be "will"?

is that something we should change?

@grantfitzsimmons